src/Auth0Client.ts

```diff
@@ -748,9 +744,10 @@ export default class Auth0Client {
       options.timeoutInSeconds || this.options.authorizeTimeoutInSeconds;
     const codeResult = await runIframe(url, this.domainUrl, timeout);
 
-    if (stateIn !== codeResult.state) {
```

Comment:
Why do you need to remove these checks?

Reply:
That was an error on my part while I was investigating something, the intention was to put them back in.